sdks/python/setup.py

```diff
@@ -226,12 +307,6 @@ def run(self):
     ]),
     install_requires=REQUIRED_PACKAGES,
     python_requires=python_requires,
-    test_suite='nose.collector',
-    setup_requires=['pytest_runner'],
-    tests_require= [
-        REQUIRED_TEST_PACKAGES,
-        INTERACTIVE_BEAM,
```

Comment:
I would expect that removing `INTERACTIVE_BEAM` would cause some tests that depend on these package to fail. I'm guessing there aren't any tests for this yet.

Reply:
Any tests that require the interactive deps should add the 'interactive' extra.  So far I haven't run into any failures related to this.